flags.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+function realpath () {
+    echo $(cd "$(dirname "$1")"; pwd)/$(basename "$1")
+}
+
+function gccPath () {
+    echo $(cd $(dirname $(which gcc-7)) && echo $(dirname $(realpath $(readlink gcc-7))));
+}
+
+function cxxFlags () {
+    echo "-nostdinc++ -std=gnu++1z"
+    echo | g++-7 -Wp,-v -x c++ - -fsyntax-only 2>&1 | grep "^[[:space:]]" | while read path; do
+        echo "-I"$(realpath $path)" "
+    done | tr -d '\n';
+}
+
+function ldFlags () {
+    echo "-L/usr/local/Cellar/gcc/7.2.0/lib/gcc/7/"
```

Comment:
This depends on brew installing gcc at that directory. A better option i just found would be this:
```bash
echo $($GCC -print-search-dirs | grep "libraries: =" | sed "s/libraries: =/-L/" | sed "s/:/ -L/g")
```
That adds a lot of other dirs too, but they should be there anyway, so with a bit of luck, this works!
**Ps:** sorry for my bad `sed` skills

Reply:
no worries!  i was trying to figure out how to get the path programmatically and then just gave up.  I'll give your command a shot and update the script.